design/sketch.webidl

```diff
@@ -643,21 +642,21 @@ dictionary GPURenderPassColorAttachmentDescriptor {
     GPUTextureView attachment;
     GPUTextureView? resolveTarget;
 
-    GPULoadOp loadOp;
-    GPUStoreOp storeOp;
-    GPUColor clearColor;
+    GPULoadOp loadOp = "clear";
```

Comment:
not an obvious default: the user may be surprised to see the contents of a target cleared

Reply:
Agreed. I think this should probably be required for now since defaulting to "load" is more costly so there's no clear choice yet.